sinopia_client/README.md

```diff
@@ -4,8 +4,8 @@ SinopiaServer - JavaScript client for sinopia_server
 Sinopia Server. Supports CRUD, following loosely LDP, on Sinopia resources. Top level requires special authorization for creating a group's graph (i.e. PCC, Stanford, etc.).
 This SDK is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
-- API version: 3.0.0-beta3
-- Package version: 3.0.0-beta3
+- API version: 3.0.0-beta5
```

Comment:
@jmartin-sul cc: @jermnelson 
Related but not directly to this README change: I wonder if we should bump the version to 3.0.0 and leave beta releases in the past, especially since we only have one consumer and we control it. Why? Then we could pin the dependency in the editor at `'^3.0.0'` and update it without having to touch package.json. Right now, we need to edit package.json in the editor to pick up any updates to the sinopia client. A minor annoyance, I know.

But, so... is there anything we're waiting on here before saying 3.0.0 is shippable?

Reply:
i'd be fine with that.  i was trying to avoid major version churn while also being good about semver, but i agree that it doesn't make a practical difference since we're the only consumers, so i'd be happy to not be a stickler for the rules in this case.

(@mjgiarlo & @jermnelson)